src/hooks/useAccountStatus.tsx

```diff
@@ -91,9 +94,17 @@ export const useAccountStatus = (poll?: boolean) => {
   const isUnauthenticated =
     error?.response?.errors?.[0]?.errorType === "UnauthorizedException";
 
+  const userNeedsSelfConfigPermissions = error?.response?.errors?.[0]?.message
+    .toLowerCase()
+    .startsWith("forbidden: requires self_config permissions");
```

Comment:
This seems a little brittle - i.e. if we change anything about how error messages are written, it'll break this. Do we have error codes we could check?

Reply:
We do not, I'm planning to raise a ticket with Tayler and then he can remove the need for this permission (as I think an Editor should know when there is a Schema update in progress to know things might be different in a second).

Tayler's talked about adding codes - a future thing
I think as a greater UI piece we should decide what permissions are required to use it, and then I can tweak the login form to check for these - I'll raise a ticket for this also.